operations/_scripts/generate/generate_app_repo.sh

```diff
@@ -17,6 +17,9 @@ fi
 
 cp -rf "$TARGET_PATH"/* "${GITHUB_ACTION_PATH}/operations/deployment/ansible/app/${GITHUB_REPO_NAME}/"
 
-echo "Copying checked in env file from repo to Ansible deployment path"
-
-cp "$TARGET_PATH/$REPO_ENV" "${GITHUB_ACTION_PATH}/operations/deployment/ansible/repo.env"
\ No newline at end of file
+if [ -s "$TARGET_PATH/$REPO_ENV" ]; then
```

Comment:
What's `-s` doing here out of curiousity?

Reply:
If it exists and not empty